src/notificationarea.cpp

```diff
@@ -25,8 +25,9 @@
  *
  * END_COMMON_COPYRIGHT_HEADER */
 
-#include <QDesktopWidget>
+#include <QDesktopWidget> // TODO: remove it and use only QScreen as QDesktopWidget has become obsolete
```

Comment:
Please remove `QDesktopWidget` from the code! It's related to your PR, although you didn't add it.

Reply:
Ok. I thought i had to create a new PR to removed it from the parts that doesn't relate to the current "issue". I will remove it then.

Reply:
Generally, you're right about new PRs but this one is related to the changes you've made. Sorry if I created a confusion by my previous remarks!